avro-fastserde/src/test/java/com/linkedin/avro/fastserde/micro/benchmark/AvroGenericDeserializer.java

```diff
@@ -0,0 +1,68 @@
+package com.linkedin.avro.fastserde.micro.benchmark;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.avro.Schema;
+import org.apache.avro.generic.GenericDatumReader;
+import org.apache.avro.io.BinaryDecoder;
+import org.apache.avro.io.DatumReader;
+import org.apache.avro.io.DecoderFactory;
+
+
+public class AvroGenericDeserializer<V> {
```

Comment:
Why are we including this class? Only for use by the micro-benchmark? Is this necessary? Why not make the micro-benchmark use `DatumReader` directly instead?

Reply:
Those are for micro-benchmark.
Considering the benchmark could use all the functionalities, such as serialize single-object/multiple objects, and de-serialize single object/multiple objects w/o reuse, and I would like to keep those two utilities to make the benchmark test simpler.